infrastructure/database/migrations/2_state_table.up.sql

```diff
@@ -2,5 +2,5 @@ CREATE TABLE "matrix_state" (
 "bot_id" INTEGER NOT NULL CHECK (bot_id <> ''),
 "what"   TEXT NOT NULL CHECK(what <> ''),
 "id"     TEXT NOT NULL CHECK(id <> ''),
-"value"  TEXT NOT NULL CHECK(value <> '')
+"value"  TEXT NOT NULL
```

Comment:
We don't know the exact behaviour i.e. value can't be empty. It could clear an old value by emptying the previous value, hence decided not to enforce empty constraint on value.